packages/timeline-state-resolver/src/integrations/vmix/index.ts

```diff
@@ -1454,3 +1432,131 @@ export interface VMixAudioChannel {
 	meterF2: number
 	headphonesVolume: number
 }
+
+/**
+ * Handles pre-loading of inputs.
+ */
+class VMixInputHandler extends EventEmitter {
```

Comment:
This class should be moved to a separate file as this file is already _much_ too long